File: CS/DXExample.Module.Win/HidePivotGridFieldsControllerWin.cs
using DevExpress.ExpressApp.PivotChart.Win;

namespace DXExample.Module.Web {
    public class HidePivotGridFieldsControllerWeb : HidePivotGridFieldsControllerBase {
        public HidePivotGridFieldsControllerWeb() { }
        protected override void SetFieldBuider(DevExpress.ExpressApp.PivotChart.IAnalysisControl control) {
            ((AnalysisControlWin)control).FieldBuilder = new CustomPivotFieldBuilder(control);
        }
    }
}

File: VB/DXExample.Module.Win/HidePivotGridFieldsControllerWin.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp.PivotChart.Win

Namespace DXExample.Module.Web
	Public Class HidePivotGridFieldsControllerWeb
		Inherits HidePivotGridFieldsControllerBase
		Public Sub New()
		End Sub
		Protected Overrides Sub SetFieldBuider(ByVal control As DevExpress.ExpressApp.PivotChart.IAnalysisControl)
			CType(control, AnalysisControlWin).FieldBuilder = New CustomPivotFieldBuilder(control)
		End Sub
	End Class
End Namespace
